tests/tools/oesign-engine/test-engine/README.md

```diff
@@ -0,0 +1,9 @@
+# Oesign Test Engine
+
+The oesign test engine allows openenclave to validate openssl engine functionality
```

Comment:
Branding: Open Enclave rather than "openenclave", similarly OpenSSL when referring to the product rather than `openssl` used on the command line.

Reply:
done